draft-ietf-ccwg-bbr.md

```diff
@@ -1146,8 +1146,16 @@ interval between now and the next ACK.
 After each packet transmission, the sender executes the following steps:
 
 ~~~~
+   Note that this logic uses UnacknolwedgedBytes() rather that C.inflight
+   because C.inflight is an estimate based on loss detection heuristics.
+   It is important to avoid having spurious loss detection episodes cause
+   a spuriously small time interval, causing a spuriously high
+   delivery rate samples.
+
+   After each packet transmission, the sender executes the following steps:
+
   OnPacketSent(Packet P):
-    if (C.inflight == 0)
+    if (UnacknolwedgedBytes() == 0)
```

Comment:
nit: typo (sorry if that was my typo!) s/UnacknolwedgedBytes()/UnacknowledgedBytes()/